packages/fxa-payments-server/src/lib/apiClient.ts

```diff
@@ -0,0 +1,94 @@
+import { Config } from './config';
```

Comment:
This might be a weird idea, but seeing how this is shaping up as an instance that we pass around everywhere... What if we made the `apiClient` module a more proper singleton - not unlike how [the `config` module](https://github.com/mozilla/fxa/blob/master/packages/fxa-payments-server/src/lib/config.ts) works now?

That is, rather than export a class to be instantiated given `config` and `accessToken` - make `config` and `accessToken` into module-local variables and export `updateConfig` & `updateAccessToken` functions along with all the API client functions. Then, we wouldn't need to pass an `apiClient` instance around, just import methods from the module

Reply:
I like that much better👍, the way I've got it now was definitely starting to feel off.